src/components/AlbumName.jsx

```diff
@@ -0,0 +1,16 @@
+import { PropTypes } from "prop-types";
+
+export const AlbumName = ({ albumName, uri }) => {
+  return (
+    <div>
```

Comment:
let's add a className for this div, so that we can target it in css. Let's also remove the dive below since it's unecessary.

Reply:
Done!